upk2esphome/parts/key.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright (c) Kuba Szczodrzyński 2023-4-21.
```

Comment:
Update the copyright to indicate the author of this file, otherwise the PR is approved :smiley:

Reply:
yay! done :)